qutebrowser/app.py

```diff
@@ -479,6 +475,10 @@ def _init_modules(*, args):
     log.init.debug("Initializing cookies...")
     cookies.init(q_app)
 
+    log.init.debug("Initializing websettings...")
+    websettings.init(args)
+    quitter.instance.shutting_down.connect(websettings.shutdown)
```

Comment:
Can you elaborate why you moved this down? From a quick look, things still seem properly when moving it up again, and the init sequence is somewhat brittle.

Reply:
I remember specifically moving it because the `webenginesettings` module now makes use of other modules that needed to be initialized first, but now it looks like it was a confusion. I moved it below the initialization of the `cookies` module, but that's not the same `cookies` module used by `webenginesettings`. And you're right, switching it back doesn't break anything. 

Hmm, maybe there was an issue and I though I was fixing it with that; but actually I fixed it with something else and forgot to revert the first change...

I'll switch it back.